app/scripts/lib/snap-keyring/snap-keyring.ts

```diff
@@ -365,18 +400,24 @@ class SnapKeyringImpl implements SnapKeyringCallbacks {
     handleUserInput: (accepted: boolean) => Promise<void>,
     onceSaved: Promise<string>,
     accountNameSuggestion: string = '',
-    displayConfirmation: boolean = false,
+    displayConfirmation: boolean = true,
+    displayAccountNameSuggestion: boolean = true,
```

Comment:
For future reviewers: The `displayConfirmation` was wrongly set to `false`. Now with this change, all pre-installed Snaps will have to send this flag while emitting the `AccountCreated` event.

Reply:
Note: The change mentioned in the comment above was reverted because it requires a change in `snap-watch-only`. It will be handled in a separate PR once the Snap is updated.